### ISSUE TITLE
Launch modal specifying its content

### ISSUE THREAD

**Issue by radarve** [nyroDev/nyroModal#29]
Hi,

I used to use earlier nyroModal version and, one of the features that I used many times was the creation of a modal (manually) and specifying the modal content, like this:

```
$.nyroModalManual({
    content: data,
    modal: true,
});
```

Now, I'm trying to migrate to this new version, but I could not find such feature...

Is this correct? Is there any workaround to enjoy of this functionallity?

Thank you very much!

**Comment by nyroDev** [comment]
For now, there is no feature like that in version 2.
The workaround might be to use the DOM filter and to link to an invisible div that you will fill with your content.

I'm currently looking for putting back this functionality

**Comment by nyroDev** [comment]
Add data Filter. Closed by 1048e26d5f9cc4bd689674ee79f73c706cc88e76
Add $.nmData function. Closed by 1048e26d5f9cc4bd689674ee79f73c706cc88e76

**Comment by radarve** [comment]
Thanks for your response! I'll do what you suggest me!
